include/rocksdb/utilities/ldb_cmd.h

```diff
@@ -70,6 +71,10 @@ class LDBCommand {
   static const std::string ARG_BLOB_GARBAGE_COLLECTION_FORCE_THRESHOLD;
   static const std::string ARG_BLOB_COMPACTION_READAHEAD_SIZE;
   static const std::string ARG_DECODE_BLOB_INDEX;
+  static const std::string ARG_BLOB_DUMP_SHOW_KEY_TYPE;
+  static const std::string ARG_BLOB_DUMP_SHOW_BLOB_TYPE;
+  static const std::string ARG_BLOB_DUMP_SHOW_UNCOMPRESSED_BLOB_TYPE;
+  static const std::string ARG_BLOB_DUMP_SHOW_SUMMARY;
```

Comment:
Instead of introducing dedicated parameters for dumping blobs and exposing all the various modes supported by `BlobDumpTool`, we could simplify things a bit and reuse the existing options that govern whether keys and values should be printed as hex (see `--key_hex`, `--value_hex`, and `--hex` and the members `is_key_hex_` and `is_value_hex_`). For example, we could pass `kHex` as `show_key` if `is_key_hex_` is set, and `kRaw` if it is not.

Blob values are a tad more complex. We would probably want to add one new option (let's call it, say, `--dump_uncompressed_blobs`) here that controls whether the uncompressed or compressed blob should be printed. (I don't think we need both at the same time.) If this new option is set, we would pass `kHex` / `kRaw` based on `is_value_hex_` as the `show_uncompressed_blob` parameter of `BlobDumpTool::Run`; if the new option is not set, we would pass `kHex` / `kRaw` as the `show_blob` parameter.

Reply:
That makes sense, thank you for this very detailed implementation explanation. I literally just directly converted this English language into C++ :).